Python/pywarpx/picmi.py

```diff
@@ -1744,27 +1744,39 @@ def initialize_inputs(self):
         # --- Use a set to ensure that fields don't get repeated.
         fields_to_plot = set()
 
+        if pywarpx.geometry.dims == 'RZ':
+            E_fields_list = ['Er', 'Et', 'Ez']
+            B_fields_list = ['Br', 'Bt', 'Bz']
+            J_fields_list = ['Jr', 'Jt', 'Jz']
+            A_fields_list = ['Ar', 'At', 'Az']
+        else:
+            E_fields_list = ['Ex', 'Ey', 'Ez']
+            B_fields_list = ['Bx', 'By', 'Bz']
+            J_fields_list = ['Jx', 'Jy', 'Jz']
+            A_fields_list = ['Ax', 'Ay', 'Az']
         if self.data_list is not None:
             for dataname in self.data_list:
                 if dataname == 'E':
-                    fields_to_plot.add('Ex')
-                    fields_to_plot.add('Ey')
-                    fields_to_plot.add('Ez')
+                    for field_name in E_fields_list:
+                        fields_to_plot.add(field_name)
                 elif dataname == 'B':
-                    fields_to_plot.add('Bx')
-                    fields_to_plot.add('By')
-                    fields_to_plot.add('Bz')
+                    for field_name in B_fields_list:
+                        fields_to_plot.add(field_name)
                 elif dataname == 'J':
-                    fields_to_plot.add('jx')
-                    fields_to_plot.add('jy')
-                    fields_to_plot.add('jz')
+                    for field_name in J_fields_list:
+                        fields_to_plot.add(field_name.lower())
```

Comment:
What does `lower()` do here and why is it done only for `J`?

Reply:
``lower()`` changes the case of the string to all lower case.  This was done because that is how ``J`` is treated elsewhere in the input parsing.  Also, ``j`` is given in lower case in the documentation.